list

```diff
@@ -472,6 +472,7 @@ histori.ca
 hit.kg
 homes.jp
 hoo.gl
+hopp.to
```

Comment:
Hi @shaulgo,

Thanks for your contribution, would you please provide some examples of `https://hopp.to/abcde` redirection to the different URL targets?

Reply:
We have both url shorteners and landing pages in this product and using the hopp.to domain.
1. url shortener 
   format: [user].hopp.to/[customName]
   example: https://shaul.hopp.to/google2
2. landing page - 
   format: hopp.to/[user]
   example: https://www.hopp.to/shaul

Reply:
do you need any more info?

Reply:
@shaulgo thanks for the update. So looks like the exact domain `hopp.to` itself isn't a URL shortener, but the subdomains are, right? May I know the use case of being added to the list? Because I don't actually add parent domains for subdomain URL shortener actually, would like to discuss the different scenarios, and see what can I help here.

Reply:
Thanks, I think there might be a misunderstanding. 
If we look at bit.ly:
bit.ly/[userCustom]

And then look at Hopp.to. It has two products with this formats: 
hopp.to/[userCustom]
[userCustom].hopp.to/[userCustom]

As I understand it. Once you have service like url shortener. Safety/ad blocking services need to be aware of it. So they don't block the parent domain. And handle every subdomain separately. 

At first we were blocked by many services. They all fixed it. NextDns has started blocking us recently. And they referred me to you. 

Blocklists, dns and this sort is new to me. So maybe I'm getting something wrong?

Reply:
NextDNS is using this list as a whitelist/allowlist, that's not wrong.

I'm not sure if they'll use external list to unblock all subdomains, but the intention here is to list the "exact" domains.

Do you have an issue on NextDNS's GitHub already so that maybe we can discuss more details?

Reply:
how is this different from bit.ly?

Reply:
https://github.com/nextdns/metadata/issues/1209

Reply:
as I understand it, hopp.to should be whitelisted? (exactly like bit.ly)
And any specific subdomain should be blocked according to its content. (we also try to prevent this usage in the first place using google web risk).

What am I not understanding?

Reply:
If `hopp.to/[user]` is only for landing page, not a URL to shorten and redirect another URL, then it's not a match for URL shortener, maybe it should be whitelisted, but under different reason, not this one then. Please feel free to correct me if I'm wrong :)

Reply:
> So because we allow users to also change the subdomain, then it's not considered as a url shortener anymore.

I don't think so, the reason not to consider it as a URL shortener domain is only because the domain itself is not used for URL shortener, or maybe use another term "hostname" might be more precise in some cases.

> But the blocklists block the parent domain, because of the subdomains. So it doesn't add up?

NextDNS and any other DNS level blockers will use different blocklists and allowlists, which won't add up, that's correct, because they aren't meant to add up.

> I'm sure that there are other url shorteners that at least allow changing the subdomain.

For those URL shorteners that can have customized subdomains, unless they also provide the service on the same domain, not the subdomain, they are not listed here I believe.

Reply:
@shaulgo in your case, I wonder if you should directly go to those blocklists that blocked your service? You can report as false-positive with evidence and reasons.

After checking a few lists for you, I believe that https://github.com/mitchellkrogza/Phishing.Database is the right place for you, though https://github.com/RPiList/specials & https://github.com/StevenBlack/hosts also block some subdomains of hopp.to, they don't directly block hopp.to.

Reply:
BTW, I also used my tool https://github.com/PeterDaveHello/chkdomain to check it for you. Most of the Secure DNS won't block your domain:

![image](https://user-images.githubusercontent.com/3691490/222172591-d200c7ec-4cd9-4f6c-8945-1c74f9bb2469.png)

Only two commercial database considered it risky, you can also directly report to them:

- https://safeweb.norton.com/report/show?url=hopp.to
- https://siteadvisor.com/sitereport.html?url=hopp.to

Reply:
I truly appreciate your help. Thanks :)